lang/en/content.php

```diff
@@ -39,7 +39,7 @@
                 "subtitle" => "We help companies in the digital transformation of their business.",
             ],
             "tile_4" => [
-                "title" => "R&D projects",
+                "title" => "R+D projects",
```

Comment:
Why is that?

Reply:
It's like in design

Reply:
R&D and R+D are correct for, but I have impression that R&D is used more often